### PR TITLE
Fix all errors from golangci-lint command

### PR DESCRIPTION
**Pull request by arjunrn**
Fixed all errors from the `golangci-lint` command

### CODE REVIEW

File: pkg/collector/skipper_collector_test.go [hunk 1]
```diff
@@ -29,33 +28,33 @@ func TestTargetRefReplicasDeployments(t *testing.T) {
 	name := "some-app"
 	defaultNamespace := "default"
 	deployment, err := newDeployment(client, defaultNamespace, name, 2, 1)
-	require.NoError(t, err)
+	assert.NoError(t, err)
```
Comment (mikkeloscar) [inline pkg/collector/skipper_collector_test.go]: why not use `require`?

File: example/main.go [hunk 2]
```diff
@@ -3,13 +3,15 @@ package main
...
 	"fmt"
+	"log"
 	"net/http"
 	"time"
 )
 
 func metricsHandler(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(200)
-	w.Write([]byte(fmt.Sprintf(`{"queue": {"length": %d}}`, size)))
+	_, e := w.Write([]byte(fmt.Sprintf(`{"queue": {"length": %d}}`, size)))
```
Comment (mikkeloscar) [inline example/main.go]: nit: please use `err` as variable name. Normal convention :)

**Comment by arjunrn** [comment]
👍

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**